File: Backend/ToDoList/ToDoList/Entity/Task1.cs
using System.ComponentModel.DataAnnotations;

namespace ToDoList.Entity
{
    public class Task1
    {
       
            [Key]
            public int TaskId { get; set; }
            public string TaskDescription { get; set; }
            public DateTime CreatedDate { get; set; } = DateTime.Now;
            public bool IsCompleted { get; set; }
            public DateTime? CompletedDate { get; set; }
            public DateTime? DueDate { get; set; }
            public string AssignedTo { get; set; }
            public string Creator { get; set; }
        public TaskStatus Status { get; set; }

        
    }
}

File: FrontEnd/ToDoList/src/app/task1.ts
import { Taskstatus } from "./taskstatus";

export class Task1 {
    taskId:number=0;
    taskDescription:string="";
    createdDate: Date =new Date();
    isCompleted:Boolean = false;
    completedDate:Date =new Date();
    dueDate :Date=new Date();
    assignedTo:string="";
    creator:string="";
    status:Taskstatus=Taskstatus.New
}
